packages/strapi-generate-new/files/config/environments/development/response.json

```diff
@@ -4,5 +4,8 @@
   },
   "responseTime": {
     "enabled": false
+  },
+  "poweredBy": {
+    "enabled": true
```

Comment:
You have to set the `value` key with `Strapi <strapi.io>`
If there is no value we have an empty input.

![screenshot 2019-01-26 at 17 17 03](https://user-images.githubusercontent.com/5716596/51789888-48aed200-218e-11e9-86ee-85a6d05a0983.png)

And it's the same for `staging` and `production` environment

Reply:
The input don't have to be empty for the end user.

Reply:
Done